docs/performance-tuning.asciidoc

```diff
@@ -175,3 +175,14 @@ Limiting the number of spans that may be recorded will reduce memory usage.
 
 Reducing max spans could result in loss of useful data about what occurred within a request,
 if it is set too low.
+
+Instead of limiting the number of spans, it can be better to drop spans with a very short duration. This will record the spans but discard them if they are very short.
+This can be implemented by providing a span-filter:
+
+```js
+const agent = require('elastic-apm-node').start({ /* configuration options */ })
+
+agent.addSpanFilter(payload => {
```

Comment:
To keep the code examples short, we prefer to not include the part about initializing the agent itself in cases like this, as one has to assume the reader already knows how to do that when reading this part of the documentation.

```suggestion
agent.addSpanFilter(payload => {
```